lookout/style/format/feature_utils.py

```diff
@@ -11,7 +11,8 @@
 
 class VirtualNode:
     def __init__(self, value: str, start: Position, end: Position,
-                 *, node: bblfsh.Node = None, y: int = None, path: str = None) -> None:
+                 *, node: bblfsh.Node = None, y: int = None, path: str = None,
+                 labeled_index: int = None) -> None:
```

Comment:
We don't need this entire change with `labeled_index`. Build your own mapping in `postprocess` and greatly simplify the code.